src/skdh/completeness/helpers.py

```diff
@@ -0,0 +1,59 @@
+import datetime
+import numpy as np
+
+def convert_sfreq_to_sampling_interval(x):
+    """
+    Convert sfreq in Hz (samples/second) to sampling interval (timedelta64[ns]).
+    :param x: np.array | float | int, sampling frequency in samples/second (Hz).
+    :return: timedelta64[ns], sampling interval as a time delta.
+    """
+    return np.timedelta64(1, 'ns') * (1 / x * 10 ** 9)
+
+
+def from_unix(ts, time_unit='s', utc_offset=0):
```

Comment:
can this not be handled by a `pandas.to_timestamp` call?

Reply:
it could be handled by pd.to_timedelta but I'm not sure if there's any benefit of using that over numpy since it returns the same data format

Reply:
sry thought you meant the np.timedelta64 function. pd.DataFrame.to_timestamp does not do the same thing.